src/main/java/gregtech/api/capability/impl/AbstractRecipeLogic.java

```diff
@@ -635,48 +629,53 @@ protected int[] calculateOverclock(@Nonnull Recipe recipe) {
         return overclockResult;
     }
 
-    /**
-     * @param recipeEUt the EU/t of the recipe attempted to be run
-     * @return true if the recipe is able to overclock, else false
-     */
-    protected boolean checkCanOverclock(int recipeEUt) {
```

Comment:
Why exactly was this removed? It seems fine to still call it after the added `modifyOverclockPre`

Reply:
This method became extraneous with the refactors, due to `getNumberOfOCs()`. The method covers the checks for `!isAllowOverclocking()`, `overclockTier <= GTValues.LV`, as well as `overclockTier > recipeTier`.